LibreNMS/Util/Url.php

```diff
@@ -85,7 +85,7 @@ public static function deviceLink($device, $text = null, $vars = [], $start = 0,
         }
 
         if ($device->location_id) {
-            $contents .= ' - ' . htmlentities($device->location);
+           // $contents .= ' - ' . htmlentities($device->location);
```

Comment:
Accidental change?

Reply:
Yes you are right, but i have a question about that, there is a specific usage on it ? 
because if i comment : 

<img width="624" alt="Capture d’écran 2019-06-16 à 19 14 31" src="https://user-images.githubusercontent.com/51302037/59567251-8b087b00-906b-11e9-9d1c-04d7081c0bb4.png">

if i uncomment : 

<img width="1052" alt="Capture d’écran 2019-06-16 à 19 16 32" src="https://user-images.githubusercontent.com/51302037/59567260-b8edbf80-906b-11e9-8873-1a1f6e4943d4.png">

problem with overlib title & location generate very large windows ...
(there is no problem with the other section is it normal ?) 

I change that ..

thanks